ios/BT/ExposureManager.swift

```diff
@@ -244,7 +241,15 @@ final class ExposureManager: NSObject {
   private var isDetectingExposures = false
 
   @discardableResult func detectExposures(completionHandler: @escaping ((ExposureResult) -> Void)) -> Progress {
-    
+    if #available(iOS 13.7, *) {
```

Comment:
I don't know anything about iOS development, but the docs here says you're supposed to set v2 in info.plist:
https://developer.apple.com/documentation/exposurenotification (under "Specify Exposure Notification API Version
")
Do you need to do this also, or should this just be done differently?
@mattThousand

Reply:
we are setting ENAPIVersion in build time according to the .env file corresponding to each HA, to me this value should be always be set in 2 now, if the user is running <13.5, ENAPIVersion is ignored, and for those running iOS 13.7+ we always want to use V2